lib/apiScenario/apiScenarioYamlLoader.ts

```diff
@@ -70,10 +71,17 @@ export class ApiScenarioYamlLoader implements Loader<RawScenarioDefinition> {
   }
 }
 
+const getAllSwaggerFilePathUnderDir = (dir: string): any[] => {
+  const allSwaggerFiles = fs
```

Comment:
We should not use `fs`, assuming apiScenario file is in local fs. The apiScenario file could be a remote uri.

Reply:
This is revert to previous version. So, how to fix it.

Reply:
there's `--specs` parameter in `run-api-scenario` command, so no need to scan swagger folders

Reply:
also, pls update the `apiScenarioSchema.ts` with the one in spec repo

Reply:
But we need to support previous version input, so leave it as what it was.